crates/bevy_sprite/src/picking_backend.rs

```diff
@@ -11,25 +11,50 @@ use bevy_ecs::prelude::*;
 use bevy_image::Image;
 use bevy_math::{prelude::*, FloatExt, FloatOrd};
 use bevy_picking::backend::prelude::*;
+use bevy_reflect::prelude::*;
 use bevy_render::prelude::*;
 use bevy_transform::prelude::*;
 use bevy_window::PrimaryWindow;
 
+/// Runtime settings for the [`SpriteBackend`].
+#[derive(Resource, Reflect)]
+#[reflect(Resource, Default)]
+pub struct SpriteBackendSettings {
+    /// When set to `true` picking will ignore any part of a sprite which is transparent
+    /// Off by default for backwards compatibility. This setting is provided to give you fine-grained
```

Comment:
It is mainly off by default for performance?

Reply:
I just saw/remembered per the linked issue @alice-i-cecile seems to suggest it is on by default with a way to turn off. As a user, I want this OOTB myself so on by default makes sense to me, for what it’s worth. To avoid churn feel free to wait for other reviews before changing your code.

Reply:
Agree, on by default makes sense as long as it isn't some big perf footgun to read the image data.

Reply:
I don't think the back compat is needed here, bevy is happy to make breaking changes between releases pre 1.0.